modules/economy-system/commands/economy-system.js

```diff
@@ -162,13 +163,13 @@ module.exports.subcommands = {
                 id: user.id
             }
         });
-        if (!balanceV) return interaction.reply(embedType(interaction.str['userNotFound']), {'%user%': `${interaction.user.username}#${interaction.user.discriminator}`});
+        if (!balanceV) return interaction.reply(embedType(interaction.str['userNotFound']), {'%user%': `${interaction.user.username}#${interaction.user.discriminator}`}, { ephemeral: true });
```

Comment:
should be part of embedType
```suggestion
        if (!balanceV) return interaction.reply(embedType(interaction.str['userNotFound'], {'%user%': `${interaction.user.username}#${interaction.user.discriminator}`}, { ephemeral: true }));
```